### PR TITLE
chor(hw6): Update MeetupAgendaItemFrom with watch for startDate

### PR DESCRIPTION
**Pull request by amalitsky**
Updated previous solution for comparison with previous `@change` approach.

The only real difference (as I see it) is the absence of local `this.previousStartsAtMs` value which is great (simpler since stateless) but is counter-weighted by the somewhat _magical_ `watch` which can be triggered from anywhere, hence might be harder to pinpoint and debug (in bigger components).

The actual `endsAt` code remained same.

### CODE REVIEW

**Comment by jsru** [comment]
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

**Comment by jsru** [comment]
Добавляю преподавателя (@ShGKme) для код-ревью.

File: 07-forms/03-MeetupAgendaItemForm/components/MeetupAgendaItemForm.vue [hunk 1]
```diff
@@ -142,6 +141,12 @@ const titleFieldLabels = {
   default: 'Нестандартный текст (необязательно)',
 };
 
+function timeStringToMs(timeStr) {
+  const [hours, minutes] = timeStr.split(':');
+
+  return (hours * 60 + Number(minutes)) * 60 * 1000;
```
Comment (amalitsky) [inline 07-forms/03-MeetupAgendaItemForm/components/MeetupAgendaItemForm.vue:147]: spent an hour on `1 + '2' => 12`

**Review by ShGKme — APPROVED** [review]

**Comment by ShGKme** [comment]
> but is counter-weighted by the somewhat magical watch which can be triggered from anywhere, hence might be harder to pinpoint and debug (in bigger components).

On the other side, it allows to describe data logic based on the data not on user interactions. But It's my "view of vue", I see you point

**Pull request merged**